src/manifests/crds.yaml

```diff
@@ -1,21 +0,0 @@
----
```

Comment:
Where is this CRD created now?

Reply:
I believe all crds are shipped with the [dex image](https://github.com/dexidp/dex/tree/v2.31.x/scripts/manifests/crds). They get created during deployment.